jest/private/jest_test.bzl

```diff
@@ -13,8 +13,15 @@ _attrs = dicts.add(js_binary_lib.attrs, {
         allow_single_file = [".js"],
         mandatory = True,
     ),
+    "sequencer": attr.label(
+        allow_single_file = True,
+        mandatory = True,
+    ),
 })
 
+def _relative_short_path(to, frm):
```

Comment:
@aspect_bazel_lib's relative_file doesn't handle these path correctly since it is from `examples/config.js` and to `../jest/sharding/js`.

Reply:
would be good to fix it there and use the shared library function

Reply:
here's the start of that fix, https://github.com/aspect-build/bazel-lib/pull/164

Reply:
@JesseTatasciore working on the aspect_bazel_lib relative_file fix; we can update this repo in a follow-up once that lands.